emhttp/plugins/dynamix.docker.manager/include/DockerClient.php

```diff
@@ -338,8 +338,12 @@ public function getAllInfo($reload=false,$com=true,$communityApplications=false)
 				$tmp['updated'] = var_export($DockerUpdate->getUpdateStatus($image),true);
 			}
 			if (!$com) $tmp['updated'] = 'undef';
-			if (empty($tmp['template']) || $reload) $tmp['template'] = $this->getUserTemplate($name);
-			if ($reload) $DockerUpdate->updateUserTemplate($name);
+			if ($ct['Manager'] !== 'dockerman')
```

Comment:
This never gets set, causing the edit issue I noticed

Reply:
This is set on line 922 by getDockerContainers()